src/components/gridView/index.tsx

```diff
@@ -38,44 +43,38 @@ export interface GridViewProps {
   /**
    * whether to keep the items initial size when orientation changes,
    * in which case the apt number of columns will be calculated automatically.
+   * Irrelevant when passing maxItemWidth
    */
   keepItemSize?: boolean;
 }
 
 interface State {
   viewWidth: number;
   numColumns: number;
+  itemSize: number;
 }
 
-
-type ExistProps = GridViewProps & State
 /**
  * @description: A auto-generated grid view that calculate item size according to given props
  * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/GridViewScreen.tsx
  */
-class GridView extends UIComponent<GridViewProps, State> {
+class GridView extends UIComponent<GridViewProps> {
   static displayName = 'GridView';
 
   static defaultProps = {
     numColumns: 3,
     itemSpacing: Spacings.s4
```

Comment:
Better take this too to const as you're using it again as default in line #136

Reply:
Done